cpp/cmake_modules/ThirdpartyToolchain.cmake

```
@@ -913,7 +928,7 @@ set(EP_COMMON_CMAKE_ARGS
     -DCMAKE_C_FLAGS_MISIZEREL=${EP_C_FLAGS_MINSIZEREL}
     -DCMAKE_C_FLAGS_RELEASE=${EP_C_FLAGS_RELEASE}
     -DCMAKE_C_FLAGS_RELWITHDEBINFO=${EP_C_FLAGS_RELWITHDEBINFO}
-    -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=${CMAKE_EXPORT_NO_PACKAGE_REGISTRY}
+    -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON
```

Revert a needless change.

```suggestion
    -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=${CMAKE_EXPORT_NO_PACKAGE_REGISTRY}
```